solidity/contracts/bridge/BridgeGovernance.sol

```diff
@@ -0,0 +1,1523 @@
+// SPDX-License-Identifier: MIT
+
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+
+pragma solidity ^0.8.9;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "./BridgeGovernanceParameters.sol";
+
+import "./Bridge.sol";
+
+/// @title Bridge Governance
+/// @notice Owns the `Bridge` contract and is responsible for updating
+///         its governable parameters in respect to governance delay individual
+///         for each parameter. The other resposibility is marking a vault
+///         address as trusted or no longer trusted.
+contract BridgeGovernance is Ownable {
+    using BridgeGovernanceParameters for BridgeGovernanceParameters.DepositData;
+    using BridgeGovernanceParameters for BridgeGovernanceParameters.RedemptionData;
+    using BridgeGovernanceParameters for BridgeGovernanceParameters.MovingFundsData;
+    using BridgeGovernanceParameters for BridgeGovernanceParameters.WalletData;
+    using BridgeGovernanceParameters for BridgeGovernanceParameters.FraudData;
+
+    BridgeGovernanceParameters.DepositData internal depositData;
+    BridgeGovernanceParameters.RedemptionData internal redemptionData;
+    BridgeGovernanceParameters.MovingFundsData internal movingFundsData;
+    BridgeGovernanceParameters.WalletData internal walletData;
+    BridgeGovernanceParameters.FraudData internal fraudData;
+
+    Bridge internal bridge;
+
+    // governanceDelays[0] -> governanceDelay
+    // governanceDelays[1] -> newGovernanceDelay
+    // governanceDelays[2] -> governanceDelayChangeInitiated
+    uint256[3] public governanceDelays;
```

Comment:
I would simply use three separate variables:
```
governanceDelay
newGovernanceDelay
governanceDelayChangeInitiated
```
instead of an array, as it's clearer and it's how we do this in other projects.

Reply:
I would do too, but the problem is contract size, and by using an array I was able to stay below the limit. If we change it to separate variables, then we go above the limit.

Reply:
Maybe add a comment about this, so that other reviewers (including the auditors) know about it?